src/components/LoginPanel.tsx

```diff
@@ -45,29 +61,30 @@ const LoginPanel: React.FC<{}> = () => {
 
     return (
         <MiddleDiv data-loaded={loaded} id="login-panel">
-            <Auth
-                supabaseClient={supabase}
-                providers={[]}
-                appearance={{
-                    theme: ThemeSupa,
-                    variables: {
-                        default: {
-                            colors: {
-                                inputText: theme.main.foreground[2],
-                                inputBackground: theme.main.background[2],
-                                inputBorder: theme.main.background[2],
-                                inputLabelText: theme.main.foreground[0],
-                                anchorTextColor: theme.main.lighterForeground[0],
-                                brand: theme.primary.background[2],
-                                brandAccent: theme.primary.background[2],
-                                brandButtonText: theme.primary.foreground[2],
-                                messageTextDanger: theme.error,
+            <Title>Login</Title>
+                <Auth
```

Comment:
can you de-indent this whole block? not sure why it is indented now - assume something to do with IDE trying to be clever!